packages/admin/admin/src/common/ClearInputAdornment.tsx

```diff
@@ -45,6 +45,10 @@ export const ClearInputAdornment = (inProps: ClearInputAdornmentProps) => {
         position,
     };
 
+    if (!hasClearableContent) {
+        return null;
+    }
```

Comment:
This change also affects all other components using it, not just `DateTimePicker` so it should have it's own changeset. 

Also, how does this affect other components? Did they have the same issue? Do they maybe already have workarounds for this?